test/e2e-tests-yaml.sh

```diff
@@ -17,8 +17,7 @@
 # This script calls out to scripts in tektoncd/plumbing to setup a cluster
 # and deploy Tekton Pipelines to it for running integration tests.
 
-go get -d github.com/GoogleCloudPlatform/cloud-builders/gcs-fetcher/cmd/gcs-fetcher
```

Comment:
Is this line no longer necessary? Was it ever?

I think I'd prefer to keep it unless we fully understand why it's here, and remove it in a separate PR.

cc @wlynch

Reply:
@ImJasonH I also have no idea why it was there. Also, as we are still using the `vendor` folder, the code is already there in the `vendor` folder :angel:

Reply:
and having this will re-download the whole `vendor/` directory.....

Reply:
well it depends on the cache state, but it will make it download stuff that are already there

Reply:
Yeah I’m not convinced we need it but I’d just rather remove it in a deliberate separate change.

Reply:
I have removed that change and separated to two commits..